Communitter/src/main/java/com/communitter/api/service/CommentService.java

```diff
@@ -0,0 +1,53 @@
+package com.communitter.api.service;
+
+import java.util.Date;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+import com.communitter.api.dto.CommentDto;
+import com.communitter.api.model.Comment;
+import com.communitter.api.model.Post;
+import com.communitter.api.model.User;
+import com.communitter.api.repository.CommentRepository;
+import com.communitter.api.repository.PostRepository;
+
+import jakarta.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class CommentService {
+    private final CommentRepository commentRepository;
+    private final PostRepository postRepository;
+    public Logger logger = LoggerFactory.getLogger(JwtService.class);
+
+    @Transactional
+    public CommentDto createComment(Long postId, CommentDto commentDto){
```

Comment:
can we add try catch block, so we can throw our exceptions we created according to the situation